onnxruntime/test/onnx/main.cc

```diff
@@ -961,6 +962,21 @@ select from 'TF8', 'TF16', 'UINT8', 'FLOAT', 'ITENSOR'. \n)");
     broken_tests.insert({"softmax_cross_entropy_input_shape_is_NCd1d2d3d4d5_none_no_weight_expanded", "DML does not support 5D+ tensors"});
     broken_tests.insert({"softmax_cross_entropy_input_shape_is_NCd1d2d3d4d5_none_no_weight_log_prob", "DML does not support 5D+ tensors"});
     broken_tests.insert({"softmax_cross_entropy_input_shape_is_NCd1d2d3d4d5_none_no_weight_log_prob_expanded", "DML does not support 5D+ tensors"});
+
+    // TODO: Remove identity tests when fixed #42638109
+    broken_tests.insert({"identity_cpu", "Optional type not yet supported for identity-16."});
```

Comment:
Hmm, shouldn't ORT just fall back to CPU in the case of unrecognized input/output types, just like for any other type (like bfloat16)?

Reply:
Btw @smk2007, I notice many disablements above that shouldn't be applicable anymore, like `broken_tests.insert({"unsqueeze_three_axes", "DML does not support 6D tensors"});`, as DML supports 8D. We should someday clean up the stale ones...

Reply:
Btw @smk2007, I notice many disablements above that shouldn't be applicable anymore, like `broken_tests.insert({"unsqueeze_three_axes", "DML does not support 6D tensors"});`, as DML supports 8D. We should someday clean up the stale ones...

Reply:
Yep, I was planning to go through them soon